### PR TITLE
Added German version of the games man page

### PR DESCRIPTION
**Pull request by kwinzig**
Resolving issue #1

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Can you translate the titles as well?
NAME, SYNOPSIS, DESCRIPTION, CONTROLS, DIRECTORIES and AUTHOR

**Comment by kwinzig** [comment]
Sure :) I'm sorry.  I must have overlooked to translate these.

File: pkg/colorful-german.man [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.\" Manpage for colorful
+.\" Contact veg@svgames.pl to correct errors or typos.
+.TH man 6 "2014-03-16" "1.2" "Games Manual"
```
Comment (suve) [inline pkg/colorful-german.man]: "Games Manual" should be translated, too. You should also update the date (as it describes the "manpage changed" date, not "project changed").

File: pkg/colorful-german.man [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+colorful - Einfacher 2D-Seitenansicht Shooter
+.SH ZUSAMMENFASSUNG
+\fBcolorful\fR
+.SH BESCHREIBUNG
+Colorful ist ein einfacher 2D Shooter, indem der Protagonist
+durch ein Labyrinth aus Höhlen und Korridoren reist und dabei farbige Artifakte einsammlet. 
+.SH STEUERUNG
+Das Spiel wird nur mit der Tastatur gespielt. Standard Steuerung:
+.TP
+\fBArrow keys\fR
```
Comment (suve) [inline pkg/colorful-german.man]: "Arrow keys" should also be translated.

**Review by suve — CHANGES_REQUESTED** [review]
Apart from those minor issues, looks good!

**Review by suve — APPROVED** [review]

**Comment by suve** [comment]
Seems fine now, merged. Thanks!

**Pull request merged**